lib/appsignal/utils.rb

```diff
@@ -49,8 +118,7 @@ def encode_utf8(value)
           )
         end
       end
-
-      extend ClassMethods
```

Comment:
Looks like this wasn't changed in an earlier pull after all? Everything is rebased on the latest develop.